app/src/main/res/values/api_keys.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Delete this file, take update from develop.
Create app/src/debug/res/values/google_maps_api.xml and add key here.